INFO     core.jenkins_utils:jenkins_utils.py:210 running cleanup
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 403 None
DEBUG    core.jenkins_utils:jenkins_utils.py:92 failed to get page on the first try, retrying after login
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /login?from=%2F HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /j_spring_security_check HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/securityRealm/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /computer/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/credentials/store/system/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /me/my-views/view/all/submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /user/admin/appearance/configSubmit HTTP/1.1" 200 133
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /user/admin/security/ HTTP/1.1" 200 None
DEBUG    conftest:conftest.py:60 Argument --remote-allow-origins=* added to the chrome
DEBUG    conftest:conftest.py:60 Argument --disable-gpu added to the chrome
DEBUG    conftest:conftest.py:60 Argument --no-sandbox added to the chrome
DEBUG    conftest:conftest.py:60 Argument --disable-dev-shm-usage added to the chrome
DEBUG    conftest:conftest.py:60 Argument --headless=new added to the chrome
DEBUG    conftest:conftest.py:60 Argument --window-size=1920,1080 added to the chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:101 Selenium Manager binary found at: /opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:114 Executing process: /opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager --browser chrome --debug --language-binding python --output json
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Found chromedriver 136.0.7103.92 in PATH: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 chrome detected at /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Running command: /usr/bin/google-chrome --version
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Output: "Google Chrome 136.0.7103.92 "
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Detected browser: chrome 136.0.7103.92
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Required driver: chromedriver 136.0.7103.113
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Driver path: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Browser path: /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.service:service.py:231 Started executable: `/usr/bin/chromedriver` in a child process with pid: 10308 using 0 to output -3
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:56899
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:56899 "POST /session HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:56899 "POST /session/50266f595845ce905d52e86102a8303b/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:56899 "GET /session/50266f595845ce905d52e86102a8303b/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:56899 "POST /session/50266f595845ce905d52e86102a8303b/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:56899 "POST /session/50266f595845ce905d52e86102a8303b/element/f.8FF88758FA0E271F67D8D5886C7EFB5D.d.1A76E85421DC4ABFBFC7A47EC3E0CFEA.e.2/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:56899 "POST /session/50266f595845ce905d52e86102a8303b/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:56899 "POST /session/50266f595845ce905d52e86102a8303b/element/f.8FF88758FA0E271F67D8D5886C7EFB5D.d.1A76E85421DC4ABFBFC7A47EC3E0CFEA.e.3/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:56899 "POST /session/50266f595845ce905d52e86102a8303b/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:56899 "POST /session/50266f595845ce905d52e86102a8303b/element/f.8FF88758FA0E271F67D8D5886C7EFB5D.d.1A76E85421DC4ABFBFC7A47EC3E0CFEA.e.6/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:56899 "GET /session/50266f595845ce905d52e86102a8303b/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:56899 "GET /session/50266f595845ce905d52e86102a8303b/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:56899 "GET /session/50266f595845ce905d52e86102a8303b/cookie HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /crumbIssuer/api/json HTTP/1.1" 200 155
DEBUG    core.jenkins_utils:jenkins_utils.py:31 update_crumb: b7ca45316bb12e616d357b948fccf6f20b0ba14291a04cc0ff34e276794d966c
DEBUG    LoginPage:login_page.py:32 login crumb: b7ca45316bb12e616d357b948fccf6f20b0ba14291a04cc0ff34e276794d966c
DEBUG    MainPage:ui_element.py:56 Click on locator ('link text', 'Manage Jenkins')
DEBUG    MainPage:ui_element.py:28 waiting for element_to_be_clickable(('link text', 'Manage Jenkins'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:56899 "POST /session/50266f595845ce905d52e86102a8303b/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:56899 "POST /session/50266f595845ce905d52e86102a8303b/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:56899 "GET /session/50266f595845ce905d52e86102a8303b/element/f.8FF88758FA0E271F67D8D5886C7EFB5D.d.78A6C8AF64C5FB35AB73B8D02C78B329.e.53/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:56899 "POST /session/50266f595845ce905d52e86102a8303b/element/f.8FF88758FA0E271F67D8D5886C7EFB5D.d.78A6C8AF64C5FB35AB73B8D02C78B329.e.53/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:56899 "GET /session/50266f595845ce905d52e86102a8303b/url HTTP/1.1" 200 0
DEBUG    ManageJenkinsPage:ui_element.py:56 Click on locator ('xpath', "//a[@href='load-statistics']/..")
DEBUG    ManageJenkinsPage:ui_element.py:28 waiting for element_to_be_clickable(('xpath', "//a[@href='load-statistics']/.."),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:56899 "POST /session/50266f595845ce905d52e86102a8303b/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:56899 "POST /session/50266f595845ce905d52e86102a8303b/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:56899 "GET /session/50266f595845ce905d52e86102a8303b/element/f.8FF88758FA0E271F67D8D5886C7EFB5D.d.1CB93811D4FCCE84080E751E700FF4BF.e.69/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:56899 "POST /session/50266f595845ce905d52e86102a8303b/element/f.8FF88758FA0E271F67D8D5886C7EFB5D.d.1CB93811D4FCCE84080E751E700FF4BF.e.69/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:56899 "GET /session/50266f595845ce905d52e86102a8303b/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:56899 "GET /session/50266f595845ce905d52e86102a8303b/title HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:56899 "DELETE /session/50266f595845ce905d52e86102a8303b HTTP/1.1" 200 0